wp-dependency-installer.php

```diff
@@ -208,10 +211,34 @@ public function apply_config() {
 				 * @param string $download_link Download link.
 				 * @param array  $dependency    Dependency configuration.
 				 */
-				$download_link = apply_filters( 'wp_dependency_download_link', $download_link, $dependency );
+				$this->config[ $slug ]['download_link'] = apply_filters( 'wp_dependency_download_link', $download_link, $dependency );
```

Comment:
There's no real performance benefit and the way it is makes the code more readable -- to me.